doc/configuration.rst

```diff
@@ -338,18 +336,26 @@ and belonging to the modules listed in ``doc_module``.
 Within your sphinx documentation ``.rst`` files, you can use easily
 add this reduced version of the Gallery. For example, the rst below adds
 the reduced version of the Gallery for ``numpy.exp``, which includes all
-examples that use the specific function ``numpy.exp``::
+examples that use the specific function ``numpy.exp``:
+
+.. code-block:: rst
 
-    .. include:: gen_modules/backreferences/numpy.exp.examples
-    .. raw:: html
+    .. minigallery:: numpy.exp
+        :add-heading:
 
-        <div class="sphx-glr-clear"></div>
+The ``add-heading`` option adds a heading for the mini-gallery, which will be a
+default generated message if no string is provided as an argument.  The level
```

Comment:
For the single-object case, I added a sentence that points back to the embedded mini-gallery so that the reader can see the default heading in action.

For the multiple-object case, I added a sentence at the end that also recommends using a custom heading.